tests/test_memory_size.cpp

```diff
@@ -109,8 +109,9 @@ TEST_P(TestBadMemorySizeFormats, rejectsBadFormats)
 
 INSTANTIATE_TEST_SUITE_P(MemorySize, TestGoodMemorySizeFormats, ValuesIn(TestGoodMemorySizeFormats::generate_args()));
```

Comment:
Could you also update `generate_args` to include some fractional sizes?

Reply:
Done